website/docs/cloud-docs/workspaces/settings/deletion.mdx

```diff
@@ -25,31 +25,60 @@ Before queuing a destroy plan, enable the **Allow destroy plans** toggle setting
 @include 'tfc-package-callouts/ephemeral-workspaces.mdx'
 <!-- END: TFC:only name:pnp-callout -->
 
-You can schedule a run to destroy all resources in a workspace at a specific
-date and time. This ephemeral workspace entitlement reduces your infrastructure
-spend by helping you automatically destroy resources that you only need
-temporarily, such as development environments.
+Configuring automatic infrastructure destruction requires admin permissions on at least the workspace.
 
-After Terraform Cloud performs an auto destroy run, it unsets the `auto-destroy` flag on the workspace. If you continue using the workspace, you can schedule another future auto destroy run to remove any new resources. 
+There are two main ways to automatically destroy a workspace's resources:
+* Schedule a run to destroy all resources in a workspace at a specific date and time.
+* Configure Terraform Cloud to destroy a workspace's infrastructure after a period of workspace inactivity.
+
+You can reduce your spending on infrastructure by automatically destroying temporary resources like development environments.
+
+After Terraform Cloud performs an auto-destroy run, it unsets the `auto-destroy-at` field on the workspace. If you continue using the workspace, you can schedule another future auto-destroy run to remove any new resources. 
 
 !> **Note:** Automatic destroy plans _do not_ prompt you for apply approval in the Terraform Cloud user interface. We recommend only using this setting for development environments.
 
-You can schedule an auto destroy run using the Terraform Cloud web UI, or the [workspace API](/terraform/cloud-docs/api-docs/workspaces).
+You can schedule an auto-destroy run using the Terraform Cloud web user interface, or the [workspace API](/terraform/cloud-docs/api-docs/workspaces).
+
+You can schedule [notifications](/terraform/cloud-docs/workspaces/settings/notifications) to alert you 12 and 24 hours before an auto-destroy run, and to report auto-destroy run results.
```

Comment:
On second read, could you move this sentence to the above paragraph? I think the two tie in together sentence tie into each other nicely:
```suggestion
You can also schedule [notifications](/terraform/cloud-docs/workspaces/settings/notifications) to alert you 12 and 24 hours before an auto-destroy run, and to report auto-destroy run results.
```